census_area/core.py

```diff
@@ -207,7 +207,7 @@ def _state_place_area(self, method, fields, state, place, year=None, return_geom
         logging.info(place['properties']['NAME'])
         place_geojson = place['geometry']
 
-        areas = method(fields, place_geojson, year, **kwargs)
+        areas = method(fields, place_geojson, year=year, **kwargs)
```

Comment:
The `supported_years` decorator [checks `kwargs` for the year, and falls back to the default year if it is not found](https://github.com/datamade/census/blob/e3fd32395ecb15eca154ff7431d3eb6463c73e77/census/core.py#L45). So, we need to pass it as a kwarg, not an arg, for it to be detected.